package.json

```diff
@@ -46,26 +46,26 @@
     "@storybook/vue3-vite": "^8.3.0",
     "@tsconfig/node20": "^20.1.4",
     "@types/jsdom": "^21.1.7",
-    "@types/node": "^20.14.5",
+    "@types/node": "^22.13.1",
     "@vitejs/plugin-vue": "^5.0.5",
-    "@vue/eslint-config-typescript": "~14.1.4",
+    "@vue/eslint-config-typescript": "~14.3.0",
     "@vue/test-utils": "^2.4.6",
-    "@vue/tsconfig": "^0.5.1",
+    "@vue/tsconfig": "^0.7.0",
     "autoprefixer": "^10.4.20",
     "eslint": "^9.15.0",
     "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-playwright": "^2.1.0",
     "eslint-plugin-storybook": "^0.11.1",
     "eslint-plugin-vue": "^9.31.0",
-    "jsdom": "^24.1.0",
-    "npm-run-all2": "^6.2.0",
+    "jsdom": "^26.0.0",
+    "npm-run-all2": "^7.0.2",
     "postcss": "^8.4.45",
     "postcss-custom-media": "^11.0.1",
     "postcss-mixins": "^11.0.1",
     "postcss-nesting": "^13.0.0",
     "storybook": "^8.3.0",
-    "typescript": "~5.4.0",
+    "typescript": "~5.7.3",
     "typescript-eslint": "^8.16.0",
     "vite": "^5.3.1",
```

Comment:
Vite v6 did throw an error when running storybook and I didn't have time to investigate. We can update Vite and Vitest in another PR if needed.